group/views.py

```diff
@@ -20,3 +23,89 @@ def create_group(request):
             return Response(serializer.data, status=status.HTTP_201_CREATED)
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
     return None
+
+
+@api_view(['POST'])
```

Comment:
I would expect this method to be an UPDATE/PATCH/PUT, because the group is already created, and you are adding a user? That means you are updating